extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -30,6 +30,8 @@ import {installCidService,} from
 import {urlReplacementsForDoc} from '../../../../src/url-replacements';
 import * as sinon from 'sinon';
 
+import {AmpDocSingle} from '../../../../src/service/ampdoc-impl';
```

Comment:
I'd rather not delay this PR anymore, so filed a bug to track moving to describes. For now, I'll go ahead and fix the tests and rebase.